subprojects/core/src/integTest/groovy/org/gradle/api/internal/tasks/userinput/UserInputHandlingIntegrationTest.groovy

```diff
@@ -437,6 +437,31 @@ class UserInputHandlingIntegrationTest extends AbstractUserInputHandlerIntegrati
         result.assertTaskSkipped(":generate")
     }
 
+    def "scan plugin can use askYesNoQuestion"() {
```

Comment:
```suggestion
    def "Can use askYesNoQuestion"() {
```
it's not really a scan plugin only thing, right?

Reply:
That is fair. The whole thing is an internal API and I think the method stayed there only because the scan plugin is using it.